db/migrate/20170410135556_add_csv_to_imports.rb

```diff
@@ -0,0 +1,6 @@
+class AddCsvToImports < ActiveRecord::Migration[5.0]
+  def change
+    add_column :imports, :csv, :string
+    remove_column :imports, :uri
```

Comment:
Re `csv`/`uri`: Good point. Both don't seem to capture what's really going on here. See below for what that field looks like. One option is to name it `file`. I'm open to ideas too.

## The raw value 

```sql
select csv from imports limit 1;
                                        csv                                         
------------------------------------------------------------------------------------
 1491845848-17177-0001-6416/burden-complement-_for-import-roster-orgs-condensed.csv
(1 row)
```
## The hydrated value

```ruby
pry(main)> Import.last.csv
  Import Load (0.5ms)  SELECT  "imports".* FROM "imports" ORDER BY "imports"."id" DESC LIMIT $1  [["LIMIT", 1]]
=> #<S3CsvUploader:0x007ff2ead8b410
 @file=
  #<CarrierWave::Storage::Fog::File:0x007ff2ead8ab00
   @base=#<CarrierWave::Storage::Fog:0x007ff2ead8b118 @uploader=#<S3CsvUploader:0x007ff2ead8b410 ...>>,
   @path="uploads/1491848362-20561-0001-2493/_for-import-roster-orgs-condensed.csv",
   @uploader=#<S3CsvUploader:0x007ff2ead8b410 ...>>,
 @model=
  #<Import:0x007ff2eadd2400
   id: 5,
   source_id: 1,
   description: ";';'",
   created_at: Mon, 10 Apr 2017 18:19:22 UTC +00:00,
   updated_at: Mon, 10 Apr 2017 18:19:23 UTC +00:00,
   transformer: "CsvTransformer",
   state: "finalizing",
   csv: "1491848362-20561-0001-2493/_for-import-roster-orgs-condensed.csv">,
 @mounted_as=:csv,
 @storage=
  #<CarrierWave::Storage::Fog:0x007ff2ead8b118 @uploader=#<S3CsvUploader:0x007ff2ead8b410 ...>>,
 @versions={}>
```